Refactoring the homepage and adding Multinet project subpage

pranavguru opened the pull request:
- Move most of the content to a "Multinet" subpage
   - Modify content based on latest release 
- Add indexing cards to navigate to different releases/efforts
- Message about collaboration and information to reach out
- Updated news section

Locke0 commented:
Would it be better if we remove "learn more" button in the release card and make the release card clickable?

harshsikka commented:
This is **great** work. Some targeted feedback:

- blurb can be much smaller, i.e. 2-3 sentences. we ofc can refactor this
- order suggestion: quick blurb, then immediately into tiles, then news, then visual abstract (the scrolling video)
- the cards are excellent. 1 suggestion - make them wider/bigger. Gives more room for content, especially on the flip effect, where it is cramped. 
- we likely will want to iterate on the CTA, need to think about this

harshsikka commented on index.html:
```diff
@@ -43,264 +53,728 @@
+                  <div class="flip-card-back">
+                    <div class="scrollable-content">
+                      <div> <!-- Content wrapper -->
+                        <h3 class="title is-4">Multiet v0.2</h3>
```
typo

harshsikka reviewed (changes requested):
This is great work. Some targeted feedback:

blurb can be much smaller, i.e. 2-3 sentences. we ofc can refactor this
order suggestion: quick blurb, then immediately into tiles, then news, then visual abstract (the scrolling video)
the cards are excellent. 1 suggestion - make them wider/bigger. Gives more room for content, especially on the flip effect, where it is cramped.
we likely will want to iterate on the CTA, need to think about this (can break this out into its own issue)

pranavguru commented:
Resolved all the above comments:
- Removed learn more from the cards and made the cards themselves clickable
- Fixed typo
- Made blurb smaller
- Changed order of components to: blurb --> tiles --> news --> visual abstract
- Made cards wider/square
